docs/content.md

```diff
@@ -442,3 +445,64 @@ When updating not just the text content within the file, but the actual list of
 - Export the XLSX to CSV with filename `internship_providers.csv`
 - Place it in the `lib/tasks/support` directory.
 - Run `bundle exec rake teaching_internship_providers:generate`.
+
+
+## Creating a new page
+
+1. **Make sure you start by creating a new branch on master**
```

Comment:
Do we have anything in the guidance on how to create a new branch? I don't think I saw anything in the other sections?